test/partitioning.jl

```diff
@@ -85,6 +85,20 @@
     p = partition(grid, BlockPartition(T(5),T(2)))
     @test length(p) == 12
     @test Set(nelements.(p)) == Set([5,10])
+
+    grid = CartesianGrid{T}(50, 50, 50)
+
+    p = partition(grid, BlockPartition(T(1.), T(1.), T(1.), neighbors = false))
+    @test length(p) == 125000
+    @test Set(nelements.(p)) == Set(1)
+    @test metadata(p) == Dict{Any,Any}()
+
+    p = partition(grid, BlockPartition(T(5.), T(5.), T(5.), neighbors = true))
+    @test length(p) == 1000
+    @test Set(nelements.(p)) == Set(125)
+    n = metadata(p)[:neighbors]
+    @test length(n) == length(p)
+    @test all(0 .< length.(n) .<= 125)
```

Comment:
The maximum number of adjacent blocks is 3x3x3=27 right? Can you please adjust the test? Also can you use the LaTeX symbol `\le` instead of `<=`?

Reply:
Oh I am so sorry, I don't know how I missed that. Changing it now. Also changed `<=` to ` ≤`